commons/src/main/kotlin/com/simplemobiletools/commons/activities/ManageBlockedNumbersActivity.kt

```diff
@@ -1,23 +1,52 @@
 package com.simplemobiletools.commons.activities
 
+import android.app.Activity
```

Comment:
optimize imports with ctrl + alt + o, theres too many new imports

Reply:
What's your take on the starred imports?

Reply:
I was using single name imports that's why. Do we need to add a limit in the .editorconfig? Maybe something like this
```
[*.kt]
ij_kotlin_name_count_to_use_star_import = 5
ij_kotlin_name_count_to_use_star_import_for_members = 5
```

Reply:
I use the default values everywhere to keep it simple. Anyway, added the new rules into editorconfig.